docs/deploying-site/README.md

```diff
@@ -80,11 +80,27 @@ But if you want your documentation to be accessiable to **anyone** with a github
 ```yml
 steps:
   - name: Build documentation site
-    uses: bcc-code/bcc-documentation-base@v4
+    uses: bcc-code/bcc-documentation-base@v5
     with:
       ...
       public: true
 ```  
 ::: warning <del></del>
 Be sure you are using **v3** or later of the action.
+:::
+
+#### Change authentication method for the documentation
+By default, the documentation is only accessible by logging in using a github account.  
+But if you want your documentation to be accessible using a different provider, than it can be configured using the `authentication` option in the action:
+```yml
+steps:
+  - name: Build documentation site
+    uses: bcc-code/bcc-documentation-base@v5
+    with:
+      ...
+      authentication: 'azuread'
+```  
+- Currently the only available providers are: **Github**, **AzureAD** and **BCC Portal**.  
```

Comment:
Perhaps also link to the documentation here so people can easily view the exact strings they could use.

And, very small nitpick, remove "only" in this sentence as I think it's already pretty great that you support these three providers 😄

Reply:
fixed